vsynth1-huffyuvbgra test fails when compiling with -march

Compiling and running tests with `-march=alderlake` and this is the only test to fail.

**System (please complete the following information):**

- **Platform:** Linux
- **Operating System:** NixOS
- **Virtualization**: Yes, LXC
- **Jellyfin Version:** n/a, used by immich
- **Jellyfin-FFmpeg Version:** 7.0.2-9
- **Hardware Acceleration:** CPU only
- **CPU & GPU Model**: intel 12600k

**FFmpeg Logs**
```
--- ./tests/ref/vsynth/vsynth1-huffyuvbgra      1970-01-01 00:00:01.000000000 +0000
+++ tests/data/fate/vsynth1-huffyuvbgra 2025-02-11 18:44:32.323861298 +0000
@@ -1,4 +1,4 @@
-98e1f0bd1734cfb0eb0b442259a7de32 *tests/data/fate/vsynth1-huffyuvbgra.avi
-14464170 tests/data/fate/vsynth1-huffyuvbgra.avi
-49c03ab1b73b7cd3cabc3c77a9479c9e *tests/data/fate/vsynth1-huffyuvbgra.out.rawvideo
-stddev:    3.16 PSNR: 38.12 MAXDIFF:   50 bytes:  7603200/  7603200
+a671af74c638d93f9e2416ffb7fc1b66 *tests/data/fate/vsynth1-huffyuvbgra.avi
+14475070 tests/data/fate/vsynth1-huffyuvbgra.avi
+a242b16c132393d73871564a3cfdbdda *tests/data/fate/vsynth1-huffyuvbgra.out.rawvideo
+stddev:   58.15 PSNR: 12.84 MAXDIFF:  208 bytes:  7603200/  7603200
Test vsynth1-huffyuvbgra failed. Look at tests/data/fate/vsynth1-huffyuvbgra.err for details.
make: *** [tests/Makefile:310: fate-vsynth1-huffyuvbgra] Error 1
make: *** Waiting for unfinished jobs....
```

**Additional Context**
Regular FFMpeg does pass this test on my system with `-march` set. Probably the bytediff limit needs a bump

https://github.com/NixOS/nixpkgs/issues/398625

--- a/debian/patches/0033-enable-gcc-vectorization-and-fix-cpuflags.patch
+++ b/debian/patches/0033-enable-gcc-vectorization-and-fix-cpuflags.patch
@@ -36,3 +36,39 @@ Index: FFmpeg/libavcodec/x86/cabac.h
  int get_cabac_inline_x86(CABACContext *c, uint8_t *const state)
  {
      int bit, tmp;
+Index: FFmpeg/libavcodec/huffyuvdsp.c
+===================================================================
+--- FFmpeg.orig/libavcodec/huffyuvdsp.c
++++ FFmpeg/libavcodec/huffyuvdsp.c
+@@ -16,6 +16,13 @@
+  * Foundation, Inc., 51 Franklin Street, Fifth Floor, Boston, MA 02110-1301 USA
+  */
+ 
++// GCC Vectorize with AVX will break huffyuv unit tests.
++#if defined(__GNUC__) && !defined(__clang__)
++    #if (__GNUC__ > 6)
++        #pragma GCC optimize ("no-tree-vectorize")
++    #endif
++#endif
++
+ #include <stdint.h>
+ 
+ #include "config.h"
+Index: FFmpeg/libavcodec/huffyuvenc.c
+===================================================================
+--- FFmpeg.orig/libavcodec/huffyuvenc.c
++++ FFmpeg/libavcodec/huffyuvenc.c
+@@ -28,6 +28,13 @@
+  * huffyuv encoder
+  */
+ 
++ // GCC Vectorize with AVX will break huffyuv unit tests.
++#if defined(__GNUC__) && !defined(__clang__)
++    #if (__GNUC__ > 6)
++        #pragma GCC optimize ("no-tree-vectorize")
++    #endif
++#endif
++
+ #include "config_components.h"
+ 
+ #include "avcodec.h"

